packages/core/contracts-ovm/insured-bridge/implementation/L1_BridgeRouter.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.7.6;
+
+import "@eth-optimism/contracts/libraries/bridge/OVM_CrossDomainEnabled.sol";
+
+import "./L2_BridgeDepositBox.sol";
+
+/**
+ * @notice Contract deployed on L1 that has an implicit reference to a DepositBox on L2 and provides methods for
+ * "Relayers" to fulfill deposit orders to that contract. The Relayers can either post capital to fulfill the deposit
+ * instantly, or request that the funds are taken out of the passive liquidity provider pool following a challenge period.
+ * @dev A "Deposit" is an order to send capital from L2 to L1, and a "Relay" is a fulfillment attempt of that order.
+ */
+contract BridgeRouter is OVM_CrossDomainEnabled {
+    // Finder used to point to latest OptimisticOracle and other DVM contracts.
+    address public finder;
+
+    // Links deposit contract addresses to L2 networks they are deployed on.
+    // TODO: Can we convert uint256 to uint128 or smaller based on some assumption about how large netIDs get?
+    mapping(uint256 => address) public netIdToDepositContract;
```

Comment:
Are we sure we want to do multiple `netIDs` now? otherwise, this would simply be just a 1:1 mapping if we only did one network to start with with a simple `l2DepositBox` vairable

Reply:
Ah yes I recall us reverting back to 1 deposit contract. Reverting.